logdna/utils.py

```diff
@@ -16,16 +18,18 @@ def sanitize_meta(meta):
     if keys_to_sanitize:
         for key in keys_to_sanitize:
             del meta[key]
-        meta['__errors'] = 'These keys have been sanitized: ' + ', '.join(keys_to_sanitize)
+        meta['__errors'] = 'These keys have been sanitized: ' + ', '.join(
+            keys_to_sanitize)
     return meta
 
+
 def get_ip():
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
     try:
         # doesn't even have to be reachable
         s.connect(('10.255.255.255', 1))
         ip = s.getsockname()[0]
-    except:
+    except OSError:
```

Comment:
do we have a minimum supported python3 version? looks like there were some changes around this in 3.3

Reply:
I haven't seen anything that would indicate a minimum level of support

Reply:
I don't think 3.3 is a supported version of python anymore (?)

Reply:
ah, quite a while ago. ok then

Reply:
I changed this one as a trap around `Exception` which is basically saying any error. Given that there was a change in the spec, and there is nothing else to do here, I think this is a sane thing.